internal/utils/utils.go

```diff
@@ -83,3 +87,93 @@ func ValidateEndpoint(endpoint string) error {
 
 	return nil
 }
+
+func GetChaincodeRequestMessageFromSignedProposal(signedProposal *pb.SignedProposal) (crmProtoBytes []byte, e error) {
```

Comment:
would seem more natural to have this in `internal/utils/proto.go`?

Reply:
no strong opinion, but I have moved it